src/dbup-clickhouse/dbup-clickhouse.csproj

```diff
@@ -1,23 +1,22 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <Description>DbUp makes it easy to deploy and upgrade SQL Server databases. This package adds ClickHouse support.</Description>
+    <Description>DbUp makes it easy to deploy and upgrade databases. This package adds ClickHouse support.</Description>
     <Title>DbUp ClickHouse Support</Title>
     <Company>DbUp Contributors</Company>
     <Product>DbUp</Product>
     <Copyright>Copyright © DbUp Contributors 2015</Copyright>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netstandard2.1;net462;net48;net8.0;net9.0</TargetFrameworks>
```

Comment:
Any reason we can't just target these two? It covers [all frameworks listed](https://learn.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-2-1)

```suggestion
    <TargetFrameworks>netstandard2.1;net462</TargetFrameworks>
```